cg/cli/workflow/mip_dna/store.py

```diff
@@ -34,82 +39,33 @@ def analysis(context, config_stream):
 
     if not config_stream:
         LOG.error("provide a config, suggestions:")
-        for analysis_obj in context.obj["tb_api"].analyses(status="completed", deleted=False)[:25]:
-            click.echo(analysis_obj.config_path)
+        suggested_wgs_analyses = tb_api.analyses(status="completed", type="wgs", deleted=False)[:25]
+        suggested_wes_analyses = tb_api.analyses(status="completed", type="wes", deleted=False)[:25]
+        for suggested_analyses in (suggested_wgs_analyses, suggested_wes_analyses):
+            for analysis_obj in suggested_analyses:
+                click.echo(analysis_obj.config_path)
         context.abort()
 
-    new_analysis = _gather_files_and_bundle_in_housekeeper(
-        config_stream, context, hk_api, status, tb_api
-    )
-    status.add_commit(new_analysis)
-    click.echo(click.style("included files in Housekeeper", fg="green"))
-
-
-def _gather_files_and_bundle_in_housekeeper(config_stream, context, hk_api, status, tb_api):
-    """Function to gather files and bundle in housekeeper"""
     try:
-        bundle_data = tb_api.add_analysis(config_stream)
+        new_analysis = gather_files_and_bundle_in_housekeeper(config_stream, hk_api, status,)
```

Comment:
I think we have a design decision to make here in a future PR:
- We send in the workflowas a parameter to this sub and modify calls to subsequent subroutines according to workflow
or
-  We duplicate the code and rename the sub to: gather_<WORKFLOW_NAME>_files_and_bundle_in_housekeeper. Diverging the code and workflows here already.
Think we should discuss with @patrikgrenfeldt , but I am ok with doing this in a future PR

Reply:
We should. I initially figured that RNA and DNA storing was so different that having separate modules in cg.meta.store was needed. Now it's evident we can suffice with just base.py and mip.py.

In fact, the only difference in cg/cli/workflow/mip_dna/store.py and cg/cli/workflow/mip_rna/store.py is the part where we handle the situation where no config file was given: we print a list of suggested cases to the terminal. If I run `cg workflow mip-dna store analysis /path/to/rna_case_config.yaml` it works just fine...

Reply:
Sounds excellent! Then, we have reached the desired level of generalization as cg is mip pipeline agnostic. And it should be easy to add balsamic specific code in a future balsamic.py.  Nicely done!